app/views/components/unorderedList.scala.html

```diff
@@ -0,0 +1,33 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import utils.Utils.emptyString
+
+@this()
+
+@(
+  id: Option[String] = None,
+  classes: Option[String] = None,
+  liMsgList: List[String] = List.empty,
+)
+
+@if(liMsgList.nonEmpty) {
+    <ul @{id.fold(emptyString)(id => s"id=$id")} class="@{classes.fold("govuk-list govuk-list--bullet")(classes => s"$classes")}">
+        @liMsgList.map { liMsg =>
```

Comment:
I would rename `liMsgList` to `listItems` just for better clarity

Reply:
Done